lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -317,6 +317,21 @@ public function getQgisProjectVersion()
         return $this->qgis->getQgisProjectVersion();
     }
 
+    /**
+     * Get the version of the Lizmap plugin
+     * used by the project editor on QGIS Desktop.
+     *
+     * @return string Version of the lizmap plugin
+     */
+    public function getLizmapPluginVersion()
+    {
+        if ($this->cfg->getPluginMetadata()) {
```

Comment:
avoid to call the same function twice. store the result into a variable to reuse it.